lib/relation-definition.js

```diff
@@ -2045,7 +2045,11 @@ RelationDefinition.embedsOne = function(modelFrom, modelToRef, params) {
     methods: params.methods,
   });
 
-  var opts = {type: modelTo};
+  var opts = Object.assign(
+    {},
```

Comment:
nitpick: `{},` looks like a duplicate, since `params.options && params.options.property ? params.options.property : {}` would guarantee the returned value is not `undefined` nor `null`.

Reply:
The idea behind this was that your original params do not get mutated - however I am happy to change as I see your point.